frappe/email/doctype/email_account/email_account.js

```diff
@@ -184,7 +191,18 @@ frappe.ui.form.on("Email Account", {
 				read as well as unread message from server. This may also cause the duplication\
 				of Communication (emails).");
 			frappe.confirm(msg, null, function() {
-				frm.set_value("email_sync_option", "ALL");
+				frm.set_value("email_sync_option", "UNSEEN");
+			});
+		}
+	},
+
+	warn_autoreply_on_incoming: function(frm) {
+		if (frm.doc.enable_incoming && frm.doc.enable_auto_reply && frm.doc.__islocal) {
+			var msg = __("Enabling Incoming and Auto Reply together during \
+				Email Account creation will send Auto Reply email to all emails synced initially.");
+			frappe.confirm(msg, null, function() {
+				// uncheck the field selected on "No" action
+				frm.set_value(frm.warn_ei_ar_parent, 0);
```

Comment:
I think it would be better to always disable auto reply instead of disabling the selected field, because "Enabling Incoming" supersedes "Enable Auto Reply" as an email function.

```suggestion
				frm.set_value("enable_auto_reply", 0);
				frappe.show_alert({message: __("Disabled Auto Reply"), indicator: "blue"});
```

Reply:
Ok sure, will do